### DISCUSSION TITLE
Add a favorite section to store patch when bank digging

### DISCUSSION THREAD

**Discussion by Onosendaifr** [probonopd/MiniDexed#680] [Ideas]
I'm currently exploring more than 1300 banks, and i wish i could put on the side all the sounds i like with the push of a button, to auto copy them in a favorite bank

**Comment by probonopd** [comment]
Can you explain a bit more how this should work in your opinion? Wouldn't it be easier to go through your voices on a PC, put them into banks there, and then copy the banks with your favorite sounds onto the MiniDexed microSD card?

**Reply by Onosendaifr** [reply]
Because it's too cumbersome and you need to have a PC near you. Mini Dexed is the perfect nomade solution to make music everywhere without a PC. Also, when you are in a creative flow, you don't want to switch to a PC, it will break your zone.

My approach to "Patch Digging" is the same as a DJ preparing a DJ set : when you don't perform, you spend time to select and sort tracks to play on the next gig. 

Listening and sorting 1300 banks take time, and you should be able to favorite/export/add any sound to a favorite pool for later.

One way to do this is adding a new "Add to Favorite Bank" function that could be assigned to any MIDI notes. After 31 patchs added, a new "Favorite2" bank is auto created, etc... then when choosing a bank you can access to those fav bank.

**Comment by Onosendaifr** [comment]
After 2 weeks of intensive use, i think this is my most wanted feature. I spend to much time writing the name of the bank/voice on a paper, then come back to PC, find again those sounds in my collection, extract them, create a custom bank, get the SD card, and upload everything on it. I wish i could use this time to make music :)

**Reply by Banana71** [reply]
I understand exactly what you mean. 
I have adapted my working method as follows. I no longer have any voices on the SD card. With the help of Dexed, I transfer or edit the sounds on the PC and only save performances in the miniDexed. I manage all voices with Dexed.

**Comment by erresse2** [comment]
after couple of weeks playing with this little gem, i have also the same thoughts... 

having a way to store favs when browsing performances for example ... some are very good... some are excellent (i am in love with the CRUMAR performance ❤)... but when it is time to remember the location (bank and performance n°)... having a way to store like shortcuts to favorites/prefered perfs or voices.... would be something really great... 

the number of performances and voices from dexed/DX/TX is so huge, that a way to go directly to the preferred ones would really improve the user experience. ... i was thinking of making a shortcut (double click or loooong press or specific button) and a new menu besides performances menu to go directly to the favs.

**Reply by Banana71** [reply]
> (i am in love with the CRUMAR performance ❤)

You must be a true connoisseur.

Unfortunately, freely saving performances to different banks isn't yet supported. However, you can use a PC to create a favorite performance folder on the SD card and copy your favorite performances into it.
Thanks to the new FTP support, you can also do this directly online.

I've prepared a small example for you here. [004_Favorite.zip](https://github.com/user-attachments/files/20231621/004_Favorite.zip) :smile:

```
LOAD"004-Favo*",8,1
READY
LIST
"C:\performance\004_Favorite\000001_Gospel Piano.ini"
"C:\performance\004_Favorite\000002_Crumar.ini"
"C:\performance\004_Favorite\000003_Afterthough.ini"
"C:\performance\004_Favorite\000004_Rock Piano.ini"
"C:\performance\004_Favorite\000005_Ascension.ini"
"C:\performance\004_Favorite\000006_Ceremony.ini"
"C:\performance\004_Favorite\000007_+ClaviStuff+.ini"
"C:\performance\004_Favorite\000008_Annika 7.ini"
"C:\performance\004_Favorite\000009_Petersdom.ini"
"C:\performance\004_Favorite\000010_Slow Ensemble.ini"
"C:\performance\004_Favorite\000011_Mello-Choir.ini"
"C:\performance\004_Favorite\000012_Mama.ini"
"C:\performance\004_Favorite\000013_Wilkinson.ini"
"C:\performance\004_Favorite\000014_SoftTube Piano.ini"
"C:\performance\004_Favorite\000015_Hatters-Mind.ini"
"C:\performance\004_Favorite\000016_Nylon-Dreams.ini"
"C:\performance\004_Favorite\000017_Fantasia.ini"
"C:\performance\004_Favorite\000018_KOKUBO '71.ini"
"C:\performance\004_Favorite\000019_Poly-SSM2040.ini"
"C:\performance\004_Favorite\000020_ - Sextet -.ini"
```

**Reply by erresse2** [reply]
yes, it is a palliative solution... and quite brutal (i need to note down the bank + perf n° when browsing, not so user friendly)..
but if one day, someone has an idea to make it more elegant... some will surely appreciate :-)

**Reply by erresse2** [reply]
> > (i am in love with the CRUMAR performance ❤)
> 
> I've prepared a small example for you here. [004_Favorite.zip](https://github.com/user-attachments/files/20231621/004_Favorite.zip) 😄
> 
> ```
> LOAD"004-Favo*",8,1
> READY
> LIST
> "C:\performance\004_Favorite\000001_Gospel Piano.ini"
> "C:\performance\004_Favorite\000002_Crumar.ini"
...
> "C:\performance\004_Favorite\000020_ - Sextet -.ini"
> ```



WHAT ??:: you didn't put CRUMAR in #1 position ??!!  :-) shame :-D

**Comment by probonopd** [comment]
I am not even sure whether the concept of having 32 voices in banks is terribly helpful anymore. I think it is a relict from the old days?
